src/platform/apollolake/include/platform/shim.h

```diff
@@ -158,6 +158,19 @@
 
 #define SHIM_L2_MECS		(SHIM_BASE + 0xd0)
 
+#define DSP_INIT_LPGPDMA(x)	(0x1110 + (2 * x))
+#define LPGPDMA_CTLOSEL_FLAG	(1 << 15)
+#define LPGPDMA_CHOSEL_FLAG	(0xFF)
+
+#define DSP_INIT_IOPO		0x1118
```

Comment:
nitpick, I'm not sure this is the register name in the HAS ? We try and follow the naming in any HW docs to allow easier lookup.